Have re-organized the Helm chart repo. (#1)
lightbend/cloudflow-helm-charts: cloudflow/README.md

# Cloudflow 

Cloudflow enables you to quickly develop, orchestrate, and operate distributed streaming applications on Kubernetes. With Cloudflow, streaming applications are comprised of small composable components wired together with schema-based contracts. Cloudflow can dramatically accelerate streaming application development—​reducing the time required to create, package, and deploy—​from weeks to hours.

> More information about Cloudflow can be found here:

Reviewer: ```suggestion
More information about Cloudflow can be found here:
- https://cloudflow.io
```

> https://www.lightbend.com/cloudflow-by-lightbend

Reviewer: ```suggestion
- https://www.lightbend.com/cloudflow-by-lightbend
```

> ## Pre-requisites

Reviewer: ```suggestion
## Prerequisites
```

- Helm, version 3 or higher.

> - Kafka cluster

Reviewer: Not needed? I would remove it from prerequisites

Reviewer: (it is not a prereq for the helm charts themselves)

> - Kafka cluster

Reviewer: ```suggestion
```

For evaluation purposes, we recommend using the Strimzi Kafka operator. If Strimzi is used, Cloudflow can create Kafka topics automatically.

https://strimzi.io

> For external Kafka clusters, the user will have to create the application topics themselves before deploying the application.

Reviewer: ```suggestion
For external Kafka clusters, you will have to create the application topics before deploying the application.
```

## Installing

> To install the Cloudflow, execute the following commands:

Reviewer: ```suggestion
Execute the following commands to install Cloudflow:
```
